decidim-core/app/packs/src/decidim/redesigned_input_character_counter.js

```diff
@@ -269,7 +268,7 @@ export default class InputCharacterCounter {
       if (remaining === 1) {
         message = MESSAGES.charactersLeft.one;
       }
-      this.$input[0].dispatchEvent(
+      this.$userInput[0].dispatchEvent(
```

Comment:
please see comment in : decidim-core/app/packs/src/decidim/input_character_counter.js

Reply:
Same as above.